.pre-commit-config.yaml

```diff
@@ -2,12 +2,14 @@
 # See http://pre-commit.com/hooks.html for more hooks
 repos:
 - repo: git://github.com/antonbabenko/pre-commit-terraform
-  rev: v1.7.0
+  rev: v1.7.1
+  sha: 091f8b15d7b458e5a0aca642483deb2205e7db02
```

Comment:
You can run `pre-commit autoupdate` to get it updated. `sha` is not required.

Reply:
hmmm maybe my `pre-commit` is out of date. It was upset at me. Will update.